frontend/wallet/src/render/components/SettingsPage.re

```diff
@@ -24,9 +88,25 @@ let make = () => {
          | Loading => React.string("...")
          | Error(err) => React.string(err##message)
          | Data(data) =>
-           <span className=Theme.Text.Body.regular>
-             {React.string("Wallet version: " ++ data##version)}
-           </span>
+           <>
+             <span className=Theme.Text.Body.regular>
+               {React.string("Wallet version: " ++ data##version)}
+             </span>
+             <div className=Styles.walletItemContainer>
+               {data##ownedWallets
```

Comment:
curious - do we ever type the payload for requests? Eg. for SettingsQueryString, we know the shape of the JSON response, so we can make sure that a property that's not in there is not trying to be accessed.  Would `@bsRecord` work for this?

Reply:
All the payloads actually do get typechecked already, happy to explain more in person :)